src/views/SslList.vue

```diff
@@ -285,8 +288,11 @@ export default defineComponent({
         {
           title: 'GCP',
           fieldNames: ['links'],
-          displayFunction: (item) => {
-            return item?.links?.provider === 'gcp' ? 'true' : 'false'
+          displayFunction: (item: Certificate) => {
+            const isGCP = _.some(item?.links, (link: Link) => {
+              return link?.provider === 'gcp' ? true : false
```

Comment:
return statement can be simplified

```suggestion
              return link?.provider === 'gcp'
```

Reply:
Done